package.json

```diff
@@ -9,10 +9,16 @@
     "url": "git://github.com/woocommerce/woocommerce-gateway-payfast.git"
   },
   "devDependencies": {
+    "@woocommerce/dependency-extraction-webpack-plugin": "^1.4.0",
+    "@woocommerce/eslint-plugin": "^1.1.0",
+    "@wordpress/scripts": "^13.0.2",
     "node-wp-i18n": "~1.2.3"
   },
   "scripts": {
-    "build": "npm run makepot",
+    "build": "npm run makepot && npm run build:webpack",
+    "build:webpack": "wp-scripts build client/blocks/index.js",
```

Comment:
I wonder if we should have a `rimraf build/*` here as well?

Reply:
Yeah, makes sense! Added in e6eb025759dc23b21dd8d83d700380230d38ae8c.